checkov/secrets/scan_git_history.py

```diff
@@ -145,13 +147,18 @@ def _get_commits_diff(self, root_folder: str, last_commit_sha: Optional[str] = N
                     curr_diff.add_file(filename=file_path, commit_diff=base_diff_format + file_diff.diff.decode('utf-8'))
                 if not curr_diff.is_empty():
                     self.commits_diff.append(curr_diff)
-            except Exception as e:
-                logging.warning(f"got error while getting commits diff, iteration: {previous_commit_idx}, error: {e}")
+            except TimeoutException:
+                logging.error(f"stopped while getting commits diff, iteration: {previous_commit_idx}")
+                return []
```

Comment:
why not return the `return self.commits_diff` ?

Reply:
because we won't use it for anything
also if it's in self, we don't actually need to return it

Reply:
there is 1 function that uses the assignment of the variable (a workaround for tests, so we do need to return it if we want it.
but also, this is what Taylor wrote in the readme of git history today:
Use `--secrets-history-timeout` to set how long the secrets scan will run on history before stopping and returning results. The default is `12h`.

so I think we want all the results until the timeout.